ios/Podfile

```diff
@@ -21,6 +21,8 @@ target 'edge' do
   # you should disable the next line.
   use_flipper!()
 
+  pod 'react-native-in-app-review', :path => '../node_modules/react-native-in-app-review'
+
```

Comment:
This should not be needed thanks to autolinking. Can you delete it and check that the feature still works?